src/components/chatbox/Markdown.tsx

```diff
@@ -19,6 +20,8 @@ export default function Markdown({ markdown, className }: MarkdownProps) {
 				p: (props) => (
 					<Text /* Body/200/Medium */ size={200} weight="medium" {...props} />
 				),
+				// @ts-expect-error - ignore type mismatch
+				a: MdxA,
```

Comment:
![CleanShot 2023-09-29 at 17 11 46@2x](https://github.com/hashicorp/dev-portal/assets/26389321/13160093-a970-48e8-811c-d33137d44c23)

Reply:
I believe you could just use `InlineLink` here since you aren't using the anchor links plugin.